src/cmd/tools/m3ctl/yaml/dbCreate.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
Can you rename this filename to `db_create.proto`?  All our file names are lower case under score separated.